### ISSUE TITLE
Incompatibility with UndergroundOcean mod

### ISSUE THREAD

**Issue by WesCook** [WhiskyTangoFawks/Expedition#132]
I just tested the mod [UndergroundOcean](https://minecraft.curseforge.com/projects/undergroundocean).  What I found was that Expedition fails to decorate caves with it installed.  Ores will not generate, and moss/cobblestone isn't applied.

Sidenote: But is there a config option to reduce the number of mossy/cobble blocks Expedition generates?  I realized while testing this conflict that caves actually look really nice when cleaned up.

**Comment by WhiskyTangoFawks** [comment]
I'll look into it, but given that the 1.10 version is an alpha release, it's likely to be a problem on their end.

There aren't config options to customise caves, it's possible, but a lot of work, and I don't really have plans for it.

**Comment by WhiskyTangoFawks** [comment]
Are you sure?  I just tested it, and it works fine for me.

**Comment by WesCook** [comment]
Well that's bizarre...  I've applied the latest Expedition+Gameplay updates and now it's working again.  Not too sure what that's about.

It's possible that oregen was broken by something else and I thought it was a conflict.  I tested directly with and without UndergroundOcean being installed though, so it seemed pretty conclusive...

I suppose everything is working as intended now though, so I'll close this bug.
